src/main/java/duke/command/PrintCommand.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import duke.operations.TaskList;
+
+/**
+ * Represents a subclass of Command.
+ */
+public class PrintCommand extends Command {
+    /**
+     * A constructor for <code>PrintCommand</code>.
+     */
+    public PrintCommand() {
+        super(null, null, null);
+    }
+
+    /**
+     * Prints the tasks in the arraylist.
+     *
+     * @param taskList the tasks to be printed.
+     * @return the task to be printed out by GUI.
+     */
+    @Override
+    public String execute(TaskList taskList) {
+        String firstMessage = "Here are the tasks in your device:";
+        return firstMessage + "\n   " + taskList.toString();
```

Comment:
Would it be better to just return concat all the required messages together and return the variable? Is there a need to split it into two portion ?